projects/1.16/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+  "block.biomancy.digester": "消化器",
+  "block.biomancy.solidifier": "分泌腺",
+  "block.biomancy.decomposer": "分解器",
+  "block.biomancy.meatsoup_cauldron": "血肉池",
+  "block.biomancy.evolution_pool": "进化池",
+
+  "container.biomancy.gulge": "深邃胃囊",
+  "container.biomancy.fleshborn_chest": "血肉箱",
+  "container.biomancy.chewer": "咀嚼器",
+  "container.biomancy.digester": "消化器",
+  "container.biomancy.solidifier": "分泌腺",
+  "container.biomancy.decomposer": "分解器",
+  "container.biomancy.evolution_pool": "进化池",
+
+  "entity.biomancy.flesh_blob": "肉块",
+  "entity.biomancy.oculus_observer": "观察之眼",
+  "entity.biomancy.fleshkin": "血肉幼体",
+  "entity.biomancy.boomling": "爆虫",
+  "entity.biomancy.failed_sheep": "失败品羊",
+  "entity.biomancy.chroma_sheep": "缤纷羊",
+  "entity.biomancy.silky_wool_sheep": "丝毛羊",
+  "entity.biomancy.thick_wool_sheep": "厚毛羊",
+  "entity.biomancy.failed_cow": "失败品牛",
+  "entity.biomancy.nutrient_slurry_cow": "营养奶牛",
+  "entity.biomancy.brood_mother": "巢穴之母",
+
+  "enchantment.biomancy.climbing": "千足虫",
+  "enchantment.biomancy.bullet_jump": "跳蚤",
+  "enchantment.biomancy.attuned_bane": "共鸣克星",
+  "enchantment.biomancy.attuned_bane_of": "%1$s 的克星",
+  "enchantment.biomancy.quick_shot": "射速",
+  "enchantment.biomancy.max_ammo": "扩容",
+
+  "effect.biomancy.dread": "恐惧",
+  "effect.biomancy.ravenous_hunger": "极度饥饿",
+  "effect.biomancy.flesh_eating_disease": "坏死",
+
+  "reagent.biomancy.growth_serum": "生长血清",
+  "reagent.biomancy.rejuvenation_serum": "复原血清",
+  "reagent.biomancy.breeding_stimulant": "育种剂",
+  "reagent.biomancy.absorption_boost": "伤害吸收剂",
+  "reagent.biomancy.insomnia_cure": "失眠治疗剂",
+  "reagent.biomancy.cleansing_serum": "净化血清",
+  "reagent.biomancy.decay_agent": "坏死血清",
+  "reagent.biomancy.mutagen_serum": "诱变血清",
+  "reagent.biomancy.blood_sample": "血液样本",
+
+  "attribute.generic.attack_distance": "攻击距离",
+
+  "tooltip.biomancy.unveiling_oculi": "赋予你细致的视线，揭示未曾所见之物。你不会再被黑暗或魔法致盲。",
+  "tooltip.biomancy.fleshborn_war_axe": "一把暴脾气的斧头，当持有者的生命值低于敌人最大生命值的 75% 时，有 60% 的几率造成敌人最大生命值的 30% 的伤害。",
+  "tooltip.biomancy.fleshborn_khopesh": "一把由斧头进化而来的镰刀剑。",
+  "tooltip.biomancy.blood_curse_axe": "持有者生命值越低，攻击速度越快的斧头。",
+  "tooltip.biomancy.leech_claw": "攻击会回复造成伤害 50% 的生命。\n还会在重击时获得 15 秒的额外生命，最多可叠加 5 层。",
+  "tooltip.biomancy.long_range_claw": "有着更长攻击距离的血肉之爪，使用重击可以激活爪子，再加长攻击距离。",
+  "tooltip.biomancy.fleshborn_guan_dao": "有着极好的攻击距离和适应性。对同一个目标的反复重击会提升它对该目标的伤害。",
+  "tooltip.biomancy.single_item_bag": "可以储存大量单一物品。\n吞噬模式下，它会尝试吃掉背包中相同的所有物品\n补充模式下，它会尝试补充你主手或副手上的方块。",
+  "tooltip.biomancy.entity_storage": "可以储存一个活着的实体在其胃袋中。",
+  "tooltip.biomancy.fleshborn_pickaxe": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.fleshborn_axe": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.fleshborn_shovel": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.oculus_key": "将所有者的血清与该钥匙合成之后，即可用来锁定方块。",
+  "tooltip.biomancy.injection_device": "允许你将试剂注入生物或块中（有些试剂不能被注入任何东西中）\n潜行单击可以装填/卸载副手玻璃瓶或试管中的液体。\n未装载液体时，可以从生物中提取血样。",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "tooltip.biomancy.injection_device": "能让你将试剂强制注入生物或方块中（有些试剂不能被注入任何东西中）\n潜行状态下单击可以装填/卸载副手玻璃瓶或试管中的液体。\n未装载液体时，可以从生物中提取血样。",

```
表述遗漏。